### PR TITLE
Add support for white LED channel on PCA9632 LED controller

### PR DESCRIPTION
**Pull request by spamik**
### Requirements

PCA9632 controller, RGBW LED strip

### Description

Marlin currently support PCA9632 LED controller for driving RGB LEDs (strips). Although PCA9632 is 4 channel controller so it's suitable driving RGBW LEDs, it's not currently implemented. This patch can use as a option 4th channel for white LEDs.

### Benefits

Driving RGBW LEDs (strips) with PCA9632 controller.

### Configurations

[Configuration.zip](https://github.com/MarlinFirmware/Marlin/files/5684362/Configuration.zip)

### CODE REVIEW

File: Marlin/src/feature/leds/pca9632.cpp [hunk 1]
```diff
@@ -89,21 +92,25 @@ static void PCA9632_WriteRegister(const byte addr, const byte regadd, const byte
...
     data[1 + (PCA9632_BLU >> 1)] = vb;
   #else
-    uint8_t data[6], len = 6;
-    data[0] = regadd + (PCA9632_RED >> 1);
-    data[1] = vr;
-    data[2] = regadd + (PCA9632_GRN >> 1);
-    data[3] = vg;
-    data[4] = regadd + (PCA9632_BLU >> 1);
-    data[5] = vb;
+    PCA9632_WriteRegister(addr, regadd + (PCA9632_RED >> 1), vr);
```
Comment (rhapsodyv) [inline Marlin/src/feature/leds/pca9632.cpp:110]: why did you change to `PCA9632_WriteRegister` instead of the `data[0] = regadd + (PCA9632_RED >> 1);`, as it was before?

File: Marlin/src/feature/leds/pca9632.cpp [hunk 2]
```diff
@@ -89,21 +92,25 @@ static void PCA9632_WriteRegister(const byte addr, const byte regadd, const byte
...
+    PCA9632_WriteRegister(addr, regadd + (PCA9632_GRN >> 1), vg);
+    PCA9632_WriteRegister(addr, regadd + (PCA9632_BLU >> 1), vb);
+
+    #if ENABLED(PCA9632_RGBW)
+      PCA9632_WriteRegister(addr, regadd + (PCA9632_WHT >> 1), vw);
+    #endif
   #endif
   Wire.beginTransmission(I2C_ADDRESS(addr));
   Wire.write(data, len);
```
Comment (rhapsodyv) [inline Marlin/src/feature/leds/pca9632.cpp]: you remove the data declaration, so this code isn't compiling at all. Can you take a look and test it?

File: Marlin/src/feature/leds/pca9632.cpp (hunk 1 above)
Comment (spamik) [reply Marlin/src/feature/leds/pca9632.cpp:110]: For some reason this didn't work. It weren't work even with original code but as with default defines in this code this part is never used (as only first 3 channels are used and in this case part with auto increment is used). Even when I try use only RGB lights but I redefine some color to 4th channel, it stops working. With calling PCA9632_WriteRegister for each channel it works. Not sure why... According to some I2C examples between each register should be some verification byte or something so may this cause this issue...

File: Marlin/src/feature/leds/pca9632.cpp (hunk 2 above)
Comment (spamik) [reply Marlin/src/feature/leds/pca9632.cpp]: Ok, stupid me... I probably just did changes, compiled it, removed unnecessary data declaration and didn't test it again. Well, no just github newbie question: I can fix code but what to do next? It's possible in some way add new commit with this fix to this pull request?

Comment (ellensp) [reply Marlin/src/feature/leds/pca9632.cpp]: just edit, commit an push to your repo, it will be added to the PR automatically
